tests/cli/test_isolation.py

```diff
@@ -96,7 +94,3 @@ def test_file_modification_during_run(
 
     with output.open("r") as fp:
         assert "test" == fp.read().strip()
-
-    diff = previous.diff(client.repo.head.commit)
-    modifications = [modification for modification in diff if modification.change_type == "M"]
-    assert 0 == len(modifications)
```

Comment:
Why remove this? I think the whole point was that the change done in the line `fp.write('print("edited")')` does not get committed and this tests for that?

Reply:
Ah, good catch. Rebase went wild. I'll fix it asap.